<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="AeroButton" name="filter_6">
          <property name="text">
           <string>2k7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="AeroButton" name="filter_5">
          <property name="text">
           <string>3k1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="AeroButton" name="filter_9">
          <property name="text">
           <string>1k8</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="AeroButton" name="filter_var1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Var1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="AeroButton" name="filter_1">
          <property name="text">
           <string>5k</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="AeroButton" name="filter_3">
          <property name="text">
           <string>3k8</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="AeroButton" name="filter_4">
          <property name="text">
           <string>3k3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="AeroButton" name="filter_8">
          <property name="text">
           <string>2k1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="AeroButton" name="filter_7">
          <property name="text">
           <string>2k4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="AeroButton" name="filter_10">
          <property name="text">
           <string>1k</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="AeroButton" name="filter_2">
          <property name="text">
           <string>4k4</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="AeroButton" name="filter_var2">
          <property name="text">
           <string>Var2</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="3">
         <widget class="QSlider" name="Var2_Slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QSlider" name="Var1_Slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AeroButton</class>
   <extends>QPushButton</extends>
   <header location="global">Util/cusdr_buttons.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
